conf/defaults.yaml

```diff
@@ -259,6 +259,10 @@ topology.disruptor.batch.size: 100
 topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 topology.state.checkpoint.interval.ms: 1000
+topology.metrics.aggregate.per.worker: false
```

Comment:
@ptgoetz No worries. I didn't take your comments in that way. It was just my criticism of **current metrics API**.

Storm community discussed about metrics several times, and the only one that I can get is the needs of new metrics feature. 

This is a last discussion which also contains requirements of metrics feature. I don't think we should address all of them, but addressing much of them would be great.
http://mail-archives.us.apache.org/mod_mbox/storm-dev/201605.mbox/%3CCAF5108jB=4aL0Z1qMMnEe7U4Yx_9TZjD11BJYhKtkTtVJoTdhQ@mail.gmail.com%3E

Adding fixes to current metrics seems not competitive to recent metrics from others stream frameworks, and even JStorm, which is the thing we would eventually evaluate (I already did a first pass and looks promising) and port if it's better. This patch only has a value with current metrics, and I don't want to break backward compatibility because of will-be-deprecated-and-dropped feature.